### PR TITLE
Added Fliters for Price Category & Slider for Ratings

### PR DESCRIPTION
**Pull request by nareshpoonia**
Added Fliters for Price Category & Slider for Ratings

### CODE REVIEW

File: styles/luggage.css [hunk 1]
```diff
@@ -25,3 +25,15 @@
 .fa-cart-plus:hover {
   border-radius: 50%;
 }
+
+.clear-all-btn {
+  color: blue;
```
Comment (lannisterrr) [inline styles/luggage.css]: use hex or variables for colors.

File: html/luggage.html [hunk 2]
```diff
@@ -35,8 +35,74 @@
...
+              type="range"
+              id="ratings"
+              name="ratings"
+              min="0"
+              max="5"
+            />
+            <label for="ratings"></label>
+          </div>
+          <div class="fs-8 m-8 flex justify-between">
+            <span>1</span>
```
Comment (lannisterrr) [inline html/luggage.html:102]: this should be a range between 500rs to 1000rs

File: html/luggage.html [hunk 3]
```diff
@@ -35,8 +35,74 @@
...
+            <span class="fs-8">Ratings</span>
+          </div>
+          <div class="m-8">
+            <input
+              class="ratings-slider"
+              type="range"
+              id="ratings"
+              name="ratings"
+              min="0"
+              max="5"
```
Comment (Hamdrive) [inline html/luggage.html:97]: There is a better and existing approach with HTML to render both range slider and its labels
Refer [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range)

**Review by Hamdrive — COMMENTED** [review]
Good work done, have suggested a better approach to use range input.
Will suggest to implement that,

File: styles/luggage.css (hunk 1 above)
Comment (nareshpoonia) [reply styles/luggage.css]: Changed to variable

File: html/luggage.html (hunk 3 above)
Comment (nareshpoonia) [reply html/luggage.html:97]: Thanks for sharing this, changed the html

File: html/luggage.html (hunk 2 above)
Comment (nareshpoonia) [reply html/luggage.html:102]: The slider is used for ratings, that's why it is kept from 0 to 5

**Pull request merged**